DemoApp/CallView.swift

```diff
@@ -8,16 +8,15 @@ import StreamVideoSwiftUI
 import Intents
 
 struct CallView: View {
-    
+
+    private var callId: String
     @StateObject var viewModel: CallViewModel
     
     @ObservedObject var appState = AppState.shared
     
-    init(callId: String? = nil) {
```

Comment:
Why is this not optional anymore? It should be used only for deeplinking.

Reply:
I preferred emptyness than nullability. This is aligned with the DeeplinkInfo that isn't nil but instead `.empty`. I consider it more "type-safe" and explicit but i'm happy to hear your thoughts if you prefer nullability.

Reply:
Ok, let's keep it like this then.